### PR TITLE
add dashboard site to app

### PR DESCRIPTION
**Pull request by Motzart**

### CODE REVIEW

**Comment by Motzart** [comment]
Screenshot 2020-11-03 at 15 52 17

File: app/screens/dashboard/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import styled from 'styled-components';
+
+const AnimatedIcon = styled.img`
+  display: block;
+  height: ${({ size }) => `${size}px`};
+  width: ${({ size }) => `${size}px`};
+`;
+
+const Dashboard = () => {
+  const isDarkModeOn = true;
```
Comment (IlyaVi) [inline app/screens/dashboard/Dashboard.js:15]: dev typo - isDarkModeOn from params

**Comment by avive** [comment]
guys, for the app - we don't want to include the whole upper row - it is for the web only. Please see the mock in figma for this screen. Also, dark mode should be picked up from the app settings.

**Comment by IlyaVi** [comment]
Screenshot 2020-11-04 at 12 33 40
@avive ui is already fixed, top screenshot showed initial state

**Review by IlyaVi — APPROVED** [review]

**Pull request merged**